msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -439,6 +364,88 @@ Form completion message appears in Adaptive Cards while sending a response to th
 
 For more information on cards and cards in bots, see [cards documentation](~/task-modules-and-cards/what-are-cards.md).
 
+## Add notifications to your message
+
```

Comment:
Would it be clearer to call out the two types of notification paths first, and then explain how they work?

"There are two ways to send a notification from your application:
* Setting `Notification.Alert` on a bot message
* Or sending an activity feed notification using the Graph API"

Reply:
Updated as mentioned.